Cargo.toml

```diff
@@ -26,8 +26,10 @@ autocfg = "1"
 
 [dependencies]
 serde = { version = "1.0.95", optional = true, default-features = false, features = ["alloc"] }
+bincode = { version = "2.0.0-rc", optional = true, default-features = false }
```

Comment:
We're not going to add a release candidate as a dependency. Upgrading to the real 2.0.0 when it is released would be a breaking change.